### ISSUE TITLE
IndexSizeError in IE11

### ISSUE THREAD

**Issue by bratukham** [IonDen/ion.sound#54]
The problem accures just before any content is rendered on the page.

![capture](https://cloud.githubusercontent.com/assets/1413357/9157154/fbe2c76e-3ef2-11e5-8f0b-6ba45c63eb97.PNG)

The problem was caught in ion.sound ver. 3.0.0 and in the last ver. (3.0.6) it's stil here.

Any ideas?

**Comment by IonDen** [comment]
Hi, please describe your environment and maybe show example page for me to test. In my tests on IE9-Edge there was no such an error.

**Comment by bratukham** [comment]
Hi, the problem was in value of **volume** variable.
As you can see from screenshot, in my case it was set to **1.1**. I don't know whether it possible or not, but any way it was so. When I set it to **1** everything starts to work fine in IE11.

**Comment by tparkin** [comment]
This problem still exists in IE 11. Setting the volume to 2.0 when loading a sound causes the error:

`
			ion.sound({
				sounds: [
						{
								name: "water_droplet"
						}
				],
				volume: 2.0,
				path: "https://s3.amazonaws.com/responcierge-assets/sounds/",
				preload: true
			});
`

**Comment by IonDen** [comment]
@tparkin, yes. IE is supporting max volume of 1.0. I'm planning to add fix to round down the volume in that case.
